mozilla-release/config/rules.mk

```diff
@@ -1201,11 +1201,22 @@ endif # SDK_BINARY
 # CHROME PACKAGING
 
 # Cliqz additional distribution files
+# For transition period (moving to new build system) let's use both, old and new
+# variant for downloading XPI.
+# NEW:
+ifdef CQZ_BUILD_ID
+CLIQZ_EXT_URL = "http://repository.cliqz.com/dist/$(MOZ_UPDATE_CHANNEL)/$(CQZ_VERSION)/$(CQZ_BUILD_ID)/cliqz@cliqz.com.xpi"
+endif  # CQZ_BUILD_ID
+
+# OLD:
+ifndef CQZ_BUILD_ID
```

Comment:
Build system firstly will upload needed extension to S3 and after this launch a build with CQZ_BUILD_ID variable, so each build can download same extension